tests/tests_all_options.yml

```diff
@@ -0,0 +1,60 @@
+---
+- name: Test we can handle all configuration options documented in manual page
+  hosts: all
+  vars:
+    ssh_c: {}
+  tasks:
+    - name: Ubuntu containers are minified, missing manual pages
+      meta: end_host
+      when:
+        - ansible_facts['distribution'] == 'Ubuntu'
+
+    - name: Make sure manual pages and bash are installed
+      package:
+        name:
+          - man
+          - bash
+        state: present
+
+    - name: Get list of options from manual page
+      shell: >-
+        set -o pipefail && man ssh_config \
+          | grep -o '^     [A-Z][A-Za-z0-9]*\(.\|  \)' \
+          | grep -v "[A-Za-z0-9] $" | grep -v "[^A-Za-z0-9 ]$" \
+          | awk '{ print $1 }' \
+          | grep -v '^$' | grep -v "^Host$" | grep -v "^Match$"
+      args:
+        executable: /bin/bash
+
+      register: ssh_options
+      changed_when: false
+
+    - debug:
+        var: ssh_options.stdout_lines
+
+    - name: Construct the configuration list
+      set_fact:
+        ssh_c: "{{ ssh_c | combine({item: 'yes'}) }}"
+      loop:
+        "{{ ssh_options.stdout_lines }}"
```

Comment:
I don't like the style of using `list = list + value` with loops - there's probably a better way to do this with some judicious combination of filters from https://docs.ansible.com/ansible/2.8/user_guide/playbooks_filters.html#

Reply:
What I do here is mostly converting a list `[a, b]` into dict `{a => 'yes', 'b' => 'yes}`. From my fast searches I found closest I can get by creating my own filter in python, which should get some performance boost, but given that this is only test, I do not think that it matters much. Do you have in mind some particular filter I could use to avoid the loop?

Reply:
I think we can leave this as it is, since this is just for testing.